orchestrator/gorc/src/commands/sign_delegate_keys.rs

```diff
@@ -46,7 +47,11 @@ impl Runnable for SignDelegateKeysCmd {
             let mut buf = bytes::BytesMut::with_capacity(size);
             prost::Message::encode(&msg, &mut buf).expect("Failed to encode DelegateKeysSignMsg!");
 
-            let signature = key.sign_ethereum_msg(&buf);
+            let data = keccak256(buf); // TODO(bolten): the rest of the orchestrator expects a hash as a message...here too?
```

Comment:
Should match behavior of 
https://github.com/PeggyJV/gravity-bridge/blob/main/module/x/gravity/keeper/msg_server.go#L80-L88 which I believe it does.

Reply:
Yeah, you also addressed this earlier responding to the open questions at the beginning, I should delete this TODO.